pkg/wasm/client.go

```diff
@@ -131,3 +131,23 @@ func (wc wasmClient) Watch(ctx context.Context, in *v1.WatchRequest, opts ...grp
 	client := v1.NewWatchServiceClient(wc.conn)
 	return client.Watch(ctx, in, opts...)
 }
+
+func (wc wasmClient) ComputablePermissions(ctx context.Context, in *v1.ComputablePermissionsRequest, opts ...grpc.CallOption) (*v1.ComputablePermissionsResponse, error) {
+	client := v1.NewSchemaServiceClient(wc.conn)
+	return client.ComputablePermissions(ctx, in, opts...)
+}
```

Comment:
These changes are bringing in the new endpoints into the wasm client. We don't actually use them yet, though.